conan/internal/util/files.py

```diff
@@ -137,9 +137,9 @@ def save_files(path, files, encoding="utf-8"):
         save(os.path.join(path, name), content, encoding=encoding)
 
 
-def load(path, encoding="utf-8"):
+def load(path, encoding="utf-8", ignore_encoding_error=False):
```

Comment:
Note sure if editing load is aceptable, maybe the check_runner should open the handle by itself if we don't want a new argument

Reply:
Yes, probably better not modify this function, ignoring encoding errors doesn't sound great for other cases (I'd also like to understand where those encoding errors came from)